yarn.lock

```diff
@@ -2,6 +2,11 @@
 # yarn lockfile v1
 
 
```

Comment:
These are a lot of changes to dependencies. I don't have a way to validate that this didn't break anything since we don't have a staging environment yet. Do you mind sharing a Loom to demo this please?

Reply:
Yeah I wasn't gonna merge this without testing locally first. I can record a video.